src/workshops/4/pgr.c

```diff
@@ -0,0 +1,53 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+enum { TWO = 2, FIVE = 5, NINE = 9 };
+
+/* Denne funktion skal returnere 1 hvis x er et primtal og 0 ellers */
```

Comment:
@MadsHeilmann https://www.doxygen.nl/manual/docblocks.html#cppblock

Reply:
There are a few different styles in the link. Which do you think we should use?

Reply:
First one